C_CPP/Templates & Libraries/readme.md

```diff
@@ -1 +0,0 @@
-
```

Comment:
- [Templates in C++](./Templates_in_CPP)

Reply:
Added Sir!